CLIC/compact/CLIC_o3_v11/BeamCal_o1_v01_01.xml

```diff
@@ -52,7 +52,7 @@
                         <shape type="Box"  dx="world_side" dy="world_side" dz="world_side"/>
                         <shape type="BooleanShape" operation="Subtraction">
                             <shape type="Tube"  rmin="BeamCal_inner_radius" rmax="BeamCal_outer_radius+env_safety" dz="BeamCal_dz+env_safety"/>
-                            <shape type="Tube"  rmin="0" rmax="3.5*mm" dz="BeamCal_dz+10*env_safety"/>
+                            <shape type="Tube"  rmin="0" rmax="3.7*mm" dz="BeamCal_dz+10*env_safety"/>
```

Comment:
Wouldn't it be better to define this variable in the global scope with a sensible name and then use in `Beampipe_o1_vo1_02.xml` and here. This is kind of a free floating number, and with time we forget what this is.